runtime/src/accounts_db.rs

```diff
@@ -691,6 +693,8 @@ pub struct AccountsDb {
 
     pub accounts_cache: AccountsCache,
 
+    sender_bg_hasher: Option<Mutex<Sender<CachedAccount>>>,
```

Comment:
I think if we switch to crossbeam channels which implement `Sync`, we can remove the `Mutex` here, i.e. like so: https://github.com/carllin/solana/commit/9d22ae16f54d1d85484e5fd848a54c595c820600

Reply:
yes, brilliant. done. I didn't like the mutex.